src/main/java/jenkins/advancedqueue/PriorityConfigurationPlaceholderTaskHelper.java

```diff
@@ -0,0 +1,31 @@
+package jenkins.advancedqueue;
+
+
+import hudson.model.Job;
+import hudson.model.Queue;
+import jenkins.advancedqueue.sorter.ItemInfo;
+import jenkins.advancedqueue.sorter.QueueItemCache;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.workflow.support.steps.ExecutorStepExecution;
+
+class PriorityConfigurationPlaceholderTaskHelper {
+
+    private static boolean placeholderTaskUsed = Jenkins.getInstance().getPlugin("workflow-durable-task-step") != null;
```

Comment:
🐛 Is not going to work if you dynamically install Pipeline, restart will be required